api/.dockerignore

```diff
@@ -0,0 +1,3 @@
+**/.env*
+**/*.pyc
+**/__pycache__/
```

Comment:
Any interest in ignoring these?
- `tests` 
-  any file named `.gitignore` (there are 2)
- `README.md`

Reply:
Since the `Dockerfile` only copies the `app` folder, `tests`, `README.md`, and `.gitignore` alongside the `Dockerfile` are all not going to be included in the container by default. However, there is a `.gitignore` in the `app` folder that would get included so I'll add an exclusion for that.